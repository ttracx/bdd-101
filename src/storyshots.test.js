import initStoryshots from '@storybook/addon-storyshots';
initStoryshots({suite: 'Image storyshots'});