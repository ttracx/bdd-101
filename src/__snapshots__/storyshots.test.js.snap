// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image storyshots MessageList A list of messages 1`] = `
<div
  className="list-group"
  data-test="message-list"
  onKeyDown={[Function]}
>
  <div
    className="list-group-item"
    data-rb-event-key={null}
    disabled={false}
    onClick={[Function]}
  >
    <span>
      Reset password request
    </span>
  </div>
  <div
    className="list-group-item"
    data-rb-event-key={null}
    disabled={false}
    onClick={[Function]}
  >
    <span>
      You have won a prize!!
    </span>
  </div>
</div>
`;

exports[`Image storyshots NotificationCentre NotificationCentre 1`] = `
<div>
  <button
    className="btn btn-primary"
    data-test="composeButton"
    disabled={false}
    type="button"
  >
    Compose
  </button>
  <div
    className="list-group"
    data-test="message-list"
    onKeyDown={[Function]}
  >
    <div
      className="list-group-item"
      data-rb-event-key={null}
      disabled={false}
      onClick={[Function]}
    >
      <span>
        1
      </span>
    </div>
    <div
      className="list-group-item"
      data-rb-event-key={null}
      disabled={false}
      onClick={[Function]}
    >
      <span>
        2
      </span>
    </div>
    <div
      className="list-group-item"
      data-rb-event-key={null}
      disabled={false}
      onClick={[Function]}
    >
      <span>
        3
      </span>
    </div>
  </div>
</div>
`;

exports[`Image storyshots Notifications A message 1`] = `
<div
  className="list-group-item"
  disabled={false}
  onClick={[Function]}
>
  <span>
    Thank you for your email
  </span>
</div>
`;

exports[`Image storyshots Notifications A message with flag 1`] = `
<div
  className="list-group-item"
  disabled={false}
  onClick={[Function]}
>
  <span>
    Important Message!
  </span>
  <span
    className="badge badge-danger"
  >
    New
  </span>
</div>
`;
